### PR TITLE
Mount tests to judge0 filesystem

### PR DESCRIPTION
**Pull request by SujithThirumalaisamy**
Updated the boilerplate generate to generate code which will fetch the inputs from a dynamic file path. 
Then Changed judge0 deployment to mount the problems directory
Resolves: Issue #35 Issue #6 
- [x] Adding dev env for project
- [x] Adding compose for judge
- [x] Migrated to yarn

**[Fix] :**
- [ ] Fix Next Auth Inside Docker ( Need Help ) 
- [ ] Need to setup a webhook in the judge0 Deployment to fetch the repo. To have the latest problems updated

### CODE REVIEW

**Review by Ankur1493 — CHANGES_REQUESTED** [review]
add this after the database url in docket-compose.yml, that might make the next-auth work, I am guessing == 
- NEXTAUTH_URL=http://localhost:3000 
- NEXTAUTH_SECRET="secret"

File: docker-compose.yml
Comment (Ankur1493) [inline docker-compose.yml:1]: in this file,

File: judge0.conf [hunk 1]
```diff
@@ -0,0 +1,357 @@
+################################################################################
+# Judge0 Configuration File
```
Comment (hkirat) [inline judge0.conf:2]: why have we added this?

Comment (SujithThirumalaisamy) [reply judge0.conf:2]: This will be moved to the ops folder and will be used as config maps

Comment (SujithThirumalaisamy) [reply judge0.conf:2]: Also useful for local dev setup. Anyway this can be there

Comment (hkirat) [reply judge0.conf:2]: Is it different from the main judge0 conf?
If yes, what is different here?

**Comment by hkirat** [comment]
can u get rid of merge conflicts?

**Pull request merged**